python/cudf/cudf/core/frame.py

```diff
@@ -41,7 +41,29 @@ def _concat(cls, objs, axis=0, ignore_index=False):
 
         # shallow-copy the input DFs in case the same DF instance
         # is concatenated with itself
-        objs = [f.copy(deep=False) for f in objs]
+        empty_counter = 0
+
+        all_empty = False
+        some_empty = False
+        result_index_length = 0
+
+        for i, obj in enumerate(objs):
+            objs[i] = obj.copy(deep=False)
+            if ignore_index:
+                # If ignore_index is true and determining if
+                # all/some objs are empty(and have index).
```

Comment:
Can we make this comment a bit more clear?

> If ignore_index is true and determining if all/some objs are empty(and have index).

then, what?

Reply:
Added more clear doc